.babelrc

```diff
@@ -88,22 +92,20 @@
         ]
       ]
     },
-    "production": {
+    "cli-production": {
```

Comment:
Why do we postfix this? Do we have a `cli-development`?

Reply:
No.
But there has to be separation between rollup and non-rollup packages for production

Reply:
@tdeekens The packages that use rollup cannot use the same presets as the ones that don't for both production and development. I'd have to rename them somehow to make it clearer.